src/gc/boehm.cr

```diff
@@ -85,6 +86,10 @@ module GC
   end
 
   def self.enable
+    unless LibGC.is_disabled != 0
+      raise "GC.enable: GC is not disabled"
```

Comment:
We should already know we're in `GC.enable` from the stacktrace. We don't prefix with the method name in other parts of the stdlib at least.

Reply:
The stack trace I get is:

```
GC is not disabled (Exception)
0x108ef5695: *CallStack::unwind:Array(Pointer(Void)) at ??
0x108ef5631: *CallStack#initialize:Array(Pointer(Void)) at ??
0x108ef5608: *CallStack::new:CallStack at ??
0x108ee4d55: *raise<Exception>:NoReturn at ??
0x108ee4d01: *raise<String>:NoReturn at ??
0x108f126ff: *GC::enable:Nil at ??
0x108ee47f4: __crystal_main at ??
0x108ef48c8: main at ??
```

Seems pretty cryptic to me. The call for `GC::enable` is like 6 lines below.

Reply:
It's a whole another issue, but it doesn't have to be this way.

Reply:
What about "Can't enable GC: GC is not disabled". I mean, there's nothing wrong with being just a bit more verbose, right?